modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewViewModel.kt

```diff
@@ -13,29 +17,41 @@ import javax.inject.Inject
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 @HiltViewModel
-class WhatsNewViewModel @Inject constructor() : ViewModel() {
+class WhatsNewViewModel @Inject constructor(
+    settings: Settings,
+    feature: FeatureWrapper,
+) : ViewModel() {
     private val _state: MutableStateFlow<UiState> = MutableStateFlow(UiState.Loading)
     val state = _state.asStateFlow()
 
     private val _navigationState: MutableSharedFlow<NavigationState> = MutableSharedFlow()
     val navigationState = _navigationState.asSharedFlow()
 
     init {
+        val isBookmarksEnabled = FeatureFlag.isEnabled(feature.bookmarksFeature)
+
         _state.value = UiState.Loaded(
-            feature = WhatsNewFeature.AutoPlay,
+            feature = if (isBookmarksEnabled) WhatsNewFeature.Bookmarks else WhatsNewFeature.AutoPlay,
+            tier = if (isBookmarksEnabled) settings.userTier else UserTier.Free,
         )
     }
 
     fun onConfirm() {
         viewModelScope.launch {
-            _navigationState.emit(NavigationState.PlaybackSettings)
+            val currentState = state.value as? UiState.Loaded ?: return@launch
+            val target = when (currentState.feature) {
+                WhatsNewFeature.AutoPlay -> NavigationState.PlaybackSettings
+                WhatsNewFeature.Bookmarks -> NavigationState.HeadphoneControlsSettings
```

Comment:
It seems a bit strange that the "Try it now" button for the bookmarks dialog goes to the headphone settings. But then again I'm not sure where it could take the user as the full screen player add bookmark action would be too tricky. 🤔

Reply:
Yeah, that's a tricky one. I asked it here p1697115334479459-slack-C055BDMU095

Reply:
I'll address the navigation in a separate PR.